dts/bindings/input/zephyr,input-lvgl-common.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Fabian Blatz <fabianblatz@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for input lvgl pseudo devices
+
+properties:
+  input:
+    required: true
```

Comment:
let's keep this optional, would be useful for situations like having multiple button devices, or even just for testing and having the application generate events